pkgs/development/compilers/bluespec-bsc/default.nix

```diff
@@ -0,0 +1,104 @@
+{ stdenv
+, fetchFromGitHub
+, fetchpatch
+, autoconf
+, automake
+, bison
+, flex
+, fontconfig
+, ghc
+, glibc
+, gmp
+, gperf
+, haskell
+, libX11
+, libpoly
+, perl
+, pkgconfig
+, verilog
+, xorg
+, zlib
+}:
+
+let
+  gmp-static = gmp.override { withStatic = true; };
```

Comment:
Why do we need static gmp?

Reply:
yices asks for it during configure phase. will add a comment.

Reply:
I feel like we should fix that too, though. Like why does *yices* need a static GMP? That seems very suspicious; I guess they could keep it as a completely internal to their `.so` (and none of the symbols get exposed by the linker)... Strange.

Reply:
yices2 in nixpkgs also uses a static gmp, so that would be something to fix on that upstream.

I'd really much prefer if we could as the yices people for a release, remove the custom vendored version from the bsc codebase, and pass in our (bumped) nixpkgs version through pkgconfig.

(Or use something like SMT-lib to avoid linking alltogether, not sure)

For the scope of the initial PR, this is probably fine.